src/sync/sync_user.hpp

```diff
@@ -100,19 +115,32 @@ friend class SyncSession;
     std::string refresh_token() const;
     State state() const;
 
+    util::AtomicSharedPtr<SyncUserContext> binding_context;
```

Comment:
Exposing a getter rather than a public member would be more consistent with the existing interface of this class.